[MI-3626]: Test cases of slash commands for MS Teams plugin (#6)
Brightscout/mattermost-test-management: data/test-cases/plugins/MS Teams sync/Slash commands/Show.md

---
# (Required) Ensure all values are filled up

> name: "Show clash command"

Reviewer: ```suggestion
name: "Show slash command"
```

status: Active
priority: Normal
folder: Slash commands
authors: "@AayushChaudhary0001"
team_ownership: 
- Change Team Name
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

1. Run the `/msteams-sync show` command in the desired MM channel.

**Step 2**

> 1. Run the `/msteams-sync show` command in the desired MM channel when the channel is not linked already.

Reviewer: ```suggestion
1. Run the `/msteams-sync show` command in the desired MM channel when the channel is not linked.
```

Reviewer: Here we are writing a different test case as "Step 2" in all the files which is misleading. We should use a title like "Test case 1", Test case 2".

Author: No we can't, as this is a standard process for all the plugins.

**Expected**

> The user should be presented with the teams channel linked with the channel in which the user is present. On step 2, the user should be presented with a message that link doesn't exist.

Reviewer 2: ```suggestion
The user should be presented with the Teams team name and channel name linked to the Mattermost channel in which the user is present. In step 2, the user should be presented with a message that the link doesn't exist.
```

**Step 1**

1. Run the "/msteams-sync show" command in the desired MM channel.

**Step 2**

1. Run the "/msteams-sync show" command in the desired MM channel when the channel is not linked already.

**Expected**

> The user should be presented with the Teams team name and channel name linked to the Mattermost channel in which the user is present. in step 2, the user should be presented with a message that the link doesn't exist.

Reviewer 2: ```suggestion
The user should be presented with the Teams team name and channel name linked to the Mattermost channel in which the user is present. In step 2, the user should be presented with a message that the link doesn't exist.
```

**Step 2**

1. Run the `/msteams-sync show` command in the desired MM channel when the channel is not linked.

**Expected**

> The user should be presented with the MS Teams team name and channel name linked to the Mattermost channel in which the user is present. In step 2, the user should be presented with a message that the link doesn't exist.

Reviewer: We should create points here corresponding to the "Steps" above.
